setup.cfg

```diff
@@ -33,7 +33,16 @@ test =
   pytest
 
 [options.entry_points]
-console_scripts = 
+console_scripts =
   pisa_analysis = pisa_utils.run:main
   covariations_int = pisa_utils.covariations_int:main
 
+[isort]
+profile = black
+line_length = 88
+multi_line_output = 3
+include_trailing_comma = True
+
+[flake8]
+max-line-length = 88
+extend-ignore = E203, W503
```

Comment:
Optional, probably best for a follow-up PR: It would be nice to ditch these and switch to ruff and uv.